### PR TITLE
LUSD AIP

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: content/aips/LUSDETHV3.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+|Risk Parameter|Value|
+| --- | --- |
+|Isolation Mode|NO|
+|Enable Borrow|YES|
+|Enable Collateral|NO|
+|Loan To Value|N/A|
+|Liquidation Threshold|N/A|
+|Liquidation Bonus|N/A|
+|Reserve Factor|10%|
+|Liquidation Protocol Fee|N/A|
```
Comment (sakulstra) [inline content/aips/LUSDETHV3.md:37]: there's a mismatch with https://github.com/aave/aip/pull/286/files#diff-783520fb0bbb52e6d3b4e4fa9140f78b41da6c003cb6e9741fefc20b00af4199R66 - i think setting to 10% is fine though.

Comment (marczeller) [reply content/aips/LUSDETHV3.md]: Fixed, thanks.

File: content/aips/RETHETHV3.md [hunk 2]
```diff
@@ -86,4 +86,8 @@ The proposal Payload was reviewed by[Bored Ghost Developing](https://bgdlabs.com
 
 ## Copyright
 
+<<<<<<< HEAD
```
Comment (sakulstra) [inline content/aips/RETHETHV3.md:89]: there should be no changes here (`git fetch origin && git checkout main content/aips/RETHETHV3.md`)

File: content/aips/LUSDETHV3.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- [tests]() // TBA
+- [proposalCode]() // TBA
+
+## Security Considerations
+
+The proposal Payload was reviewed by [Bored Ghost Developing](https://bgdlabs.com/).
+
+## Deployed Contracts
+
+- [AaveV3EthLUSDPayload: ProposalPayload]() // TBA
```
Comment (sakulstra) [inline content/aips/LUSDETHV3.md:85]:
```suggestion
- [AaveV3EthLUSDPayload: ProposalPayload](https://etherscan.io/address/0x9d4948be10dce66c0f584a6c42fd7d985786b439#code)
```

File: content/aips/LUSDETHV3.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    });
+```
+
+## References
+
+A list of relevant links like for this proposal e.g.
+
+- [forum discussion](https://governance.aave.com/t/arc-add-lusd-to-ethereum-v3-market/11522)
+- [tests](https://github.com/marczeller/aave-v3-crosschain-listing-template/blob/master/src/test/mainnet/AaveV3EthLUSDPayloadTest.t.sol)
+- [proposalCode](https://etherscan.io/address/0x9d4948be10dce66c0f584a6c42fd7d985786b439#readContract)
```
Comment (sendra) [inline content/aips/LUSDETHV3.md:77]: better to put the link to the github code here, as the link to the deployed is already on the deployed contracts section

Comment (marczeller) [reply content/aips/LUSDETHV3.md:77]: done, thx!

File: content/aips/LUSDETHV3.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    });
+```
+
+## References
+
+A list of relevant links like for this proposal e.g.
+
+- [forum discussion](https://governance.aave.com/t/arc-add-lusd-to-ethereum-v3-market/11522)
+- [tests](https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/test/mainnet/AaveV3EthLUSDPayloadTest.t.sol)
+- [proposalCode](https://etherscan.io/address/0x9d4948be10dce66c0f584a6c42fd7d985786b439#readContract)
```
Comment (sendra) [inline content/aips/LUSDETHV3.md:77]:
```suggestion
- [proposalCode](https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/contracts/mainnet/AaveV3EthLusdPayload.sol)
```

**Review by sendra — APPROVED** [review]

**Review by sakulstra — APPROVED** [review]

**Pull request merged**